### ISSUE TITLE
Tournaments standings missing color information

### ISSUE THREAD

**Issue by jonsnowpt** [Betarena/scores#979]
We need more color codes for the following leagues that are missing them:

https://scores.betarena.com/football/indonesia/liga-1
https://scores.betarena.com/football/australia/a-league
https://scores.betarena.com/football/angola/girabola
https://scores.betarena.com/football/egypt/premier-league
https://scores.betarena.com/football/congo-dr/super-ligue
https://scores.betarena.com/football/belgium/pro-league

**Comment by migbash** [comment]
**NOTES**

> `league_id: 1019` | https://scores.betarena.com/football/indonesia/liga-1
> `league_id: 1356` | https://scores.betarena.com/football/australia/a-league
> `league_id: 815` | https://scores.betarena.com/football/angola/girabola
> `league_id: 830` | https://scores.betarena.com/football/egypt/premier-league
> `league_id: 824` | https://scores.betarena.com/football/congo-dr/super-ligue
> `league_id: 208` | https://scores.betarena.com/football/belgium/pro-league

**Comment by jonsnowpt** [comment]
@migbash 

Requested, get back to you when done!

**Comment by migbash** [comment]
**NOTE**

> Updated on `DB`

**Comment by jonsnowpt** [comment]
@migbash 

Still missing here:

https://scores.betarena.com/br/futebol/portugal/league-cup
https://scores.betarena.com/football/congo-dr/super-ligue

Do we need more color variations?

**Comment by migbash** [comment]
> @jonsnowpt Those are different links to those identified, need to check and report back. But looks like yes - more color variations are needed.

> Those from this issue and #923 have been dealt with and are visible.

**Comment by jonsnowpt** [comment]
@migbash 

Yes, that was the reason for the two opened issues; they had different links. Please advise on the colors needed.

**Comment by migbash** [comment]
> @jonsnowpt both of those links [previous message](https://github.com/Betarena/scores/issues/979#issuecomment-1457760547) has the correct `color-codes` -  was just missing an update, as I updated `surgically`. I have triggered an `all` cache standings update. Will update when done so you can check for other missing `color-codes`.

**Comment by jonsnowpt** [comment]
All working.
